core/testing/build.gradle.kts

```
@@ -33,6 +33,7 @@ dependencies {
     api(libs.junit4)
     api(libs.kotlinx.coroutines.test)
     api(libs.turbine)
+    implementation(project(mapOf("path" to ":core:database")))
```

```suggestion
    implementation(project(":core:database"))
```
And move it to below to make deps list sorted as the before